tests/functional/test_benchmark_3d_case_3.py

```diff
@@ -0,0 +1,145 @@
+
+The tests check whether effective permeabilities and boundary are correctly assigned.
+Since we solved the actual model, we also implicitly check that the model does not
+crash. However, we are currently not checking that the solution is the "correct" one.
+
+"""
+import sys
+
+import numpy as np
+import pytest
+
+import porepy as pp
+
+# Append the top PorePy folder to the path, to allow for imports of the examples folder
+sys.path.append("../..")
+
+from examples.flow_benchmark_3d_case_3 import (
+    solid_constants,
+    FlowBenchmark3dCase3Model,
+)
+
+
+class ModelWithEffectivePermeability(FlowBenchmark3dCase3Model):
+    """
+    Mixin that contains the computation of effective normal permeabilities.
+    """
+
+    def effective_normal_permeability(
+            self, interfaces: list[pp.MortarGrid]
+            ) -> pp.ad.Operator:
+        """
+        Computes the effective normal permeability, see Eq. 6b from [1].
+
+        Note:
+            The effective normal permeability is the scalar that multiplies the
+            pressure jump in the continuous interface law.
+
+        Reference:
+            [1] https://doi.org/10.1016/j.advwatres.2020.103759
+
+        Parameters:
+            interfaces: List of interfaces.
+
+        Returns:
+            Wrapped ad operator containing the effective normal permeabilities for
+            the given list of interfaces.
+        """
+        subdomains = self.interfaces_to_subdomains(interfaces)
+        projection = pp.ad.MortarProjections(self.mdg, subdomains, interfaces, dim=1)
+
+        normal_gradient = pp.ad.Scalar(2) * (
+            projection.secondary_to_mortar_avg
+            @ self.aperture(subdomains) ** pp.ad.Scalar(-1)
+        )
+
+        effective_normal_permeability = (
+            self.specific_volume(interfaces)
+            * self.normal_permeability(interfaces)
+            * normal_gradient
+        )
+
+        return effective_normal_permeability
+
+
+@pytest.fixture(scope="module")
+def model() -> ModelWithEffectivePermeability:
+    """
+    Run the benchmark model with the coarsest mesh resolution.
+
+    Returns:
+        The solved model, an instance of `ModelWithEffectivePermeability`.
+
+    """
+    model = ModelWithEffectivePermeability(
+        {"material_constants": {"solid": solid_constants}}
+    )
+    pp.run_time_dependent_model(model, {})
+    return model
+
+
+@pytest.mark.skipped  # reason: slow
+def test_effective_tangential_permeability_values(model) -> None:
+    """
+    Test if the permeability values are consistent with the benchmark specification.
+
+    Parameters:
+        model: ModelWithEffectivePermeability
+            Solved model. Returned by the `model()` fixture.
+
+    """
+    for sd, d in model.mdg.subdomains(return_data=True):
+        val = np.mean(
+            d[pp.PARAMETERS][model.darcy_keyword]["second_order_tensor"].values[0][0]
+        )
+        if sd.dim == 3:
+            np.testing.assert_allclose(val, 1)
```

Comment:
Please either explain the origin of these hard-coded values or (even better) define a method effective_tangential_permeability which computes them (using Eq. 6a, right?) from the values provided in table 5.

Reply:
I did not do that because the second order tensor stored in the data dictionary is actually the effective tangential permeability (the specific volume multiplies the tangential permeability [here](https://github.com/pmgbergen/porepy/blob/e4d484f915cde39cc17639e24c1ffe5ab37c1f99/src/porepy/models/fluid_mass_balance.py#L837)).

But this is not a problem, I will add a method `effective_tangential_permeability` to the `ModelWithEffectivePermeability` class with a more in-depth explanation.

Reply:
I have added the method `effective_tangential_permeability` as explained above. Please, let me know if this is more or less what you envisioned.